src/extensions/default/SVGCodeHints/SVGUtils.js

```diff
@@ -0,0 +1,339 @@
+/*
+ * Copyright (c) 2014 Amin Ullah Khan. All rights reserved.
```

Comment:
Commits were not created until 2015, so all the copyrights should be 2015.

I'm not sure if the copyright needs to have be "Adobe Systems, Inc." in it. I need to check that.